### PR TITLE
Updated README.md (actually including a proper description of what's going on)

### PR DESCRIPTION
**Pull request by gusbocju**

### CODE REVIEW

**Review by kosara-g — APPROVED** [review]
Good job!

File: README.md [hunk 1]
```diff
@@ -18,5 +19,60 @@ Clone the repo using:
...
+### 4. Protocol Requests 
+This section describes the the the protocol requests. Fields of requests and their types are denoted: Type field name.
+
+#### 4.1 Common Requests
+
+##### Announce Presence
+This message is intended as a way of letting cars in the network know about each other. Each car should sent an Announce Presence when joining a platoon. Different channels are used to distinguish between different platoons. 
+
+***Fields***
+* unit_8_t channel - A unique identifier of the car sending the follow announcement.
```
Comment (maansaake) [inline README.md]: uint8_t written as unit_8_t, consider changing before submitting

File: README.md [hunk 2]
```diff
@@ -18,5 +19,60 @@ Clone the repo using:
...
+
+***Fields***
+* unit_8_t meantForCar - A unique identifier of the car that initiated the following.
+* unit_8_t carToFollow - A unique identifier of the final car within the platoon line.
+
+##### Emergency Brake
+This message is sent in order to stop the cars from moving. The request is used to avoid any collision and possible damage between the cars. 
+
+***Fields***
+* Identifier speed - A speed value of 0, used to stop the cars.
```
Comment (maansaake) [inline README.md]: Identifier?

File: README.md [hunk 3]
```diff
@@ -18,5 +19,60 @@ Clone the repo using:
...
+##### Follow Response
+This message is sent in response to a Follow Request. The message returns the identifier of the car at the last position of the platoon and the car ID that originally initiated the following.
+
+***Fields***
+* unit_8_t meantForCar - A unique identifier of the car that initiated the following.
+* unit_8_t carToFollow - A unique identifier of the final car within the platoon line.
+
+##### Emergency Brake
+This message is sent in order to stop the cars from moving. The request is used to avoid any collision and possible damage between the cars. 
+
```
Comment (maansaake) [inline README.md]: State if the message expects a response or not!

File: README.md [hunk 4]
```diff
@@ -18,5 +19,60 @@ Clone the repo using:
...
 
-This repo uses the license [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html). Please refer to the official documentation before you contribute.
+### 4. Protocol Requests 
+This section describes the the the protocol requests. Fields of requests and their types are denoted: Type field name.
+
+#### 4.1 Common Requests
+
+##### Announce Presence
+This message is intended as a way of letting cars in the network know about each other. Each car should sent an Announce Presence when joining a platoon. Different channels are used to distinguish between different platoons. 
+
```
Comment (maansaake) [inline README.md]: Response required?

**Review by maansaake — APPROVED** [review]
Looks good, minor changes

File: README.md (hunk 4 above)
Comment (BigPapaKlonas) [reply README.md]: no response required is now added to readme

File: README.md (hunk 3 above)
Comment (BigPapaKlonas) [reply README.md]: added

File: README.md (hunk 2 above)
Comment (BigPapaKlonas) [reply README.md]: fixed

**Pull request merged**